diff --git a/ATMHudView.m b/ATMHudView.m
index dc8d679..9498ed8 100644
--- a/ATMHudView.m
+++ b/ATMHudView.m
@@ -9,6 +9,7 @@
  *	https://github.com/atomton/ATMHud
  */
 
+#import <UIKit/UIKit.h>
 #import "ATMHudView.h"
 #import "ATMTextLayer.h"
 #import "ATMProgressLayer.h"
diff --git a/ATMProgressLayer.m b/ATMProgressLayer.m
index 5ea684d..e2e8869 100644
--- a/ATMProgressLayer.m
+++ b/ATMProgressLayer.m
@@ -9,6 +9,7 @@
  *	https://github.com/atomton/ATMHud
  */
 
+#import <UIKit/UIGraphics.h>
 #import "ATMProgressLayer.h"
 
 @implementation ATMProgressLayer
diff --git a/ATMSoundFX.m b/ATMSoundFX.m
index 70477d7..657d900 100644
--- a/ATMSoundFX.m
+++ b/ATMSoundFX.m
@@ -9,6 +9,7 @@
  *	https://github.com/atomton/ATMHud
  */
 
+#import <Foundation/Foundation.h>
 #import "ATMSoundFX.h"
 
 @implementation ATMSoundFX
@@ -51,4 +52,4 @@
     AudioServicesPlaySystemSound(_soundID);
 }
 
-@end
\ No newline at end of file
+@end
diff --git a/ATMTextLayer.m b/ATMTextLayer.m
index 9d7be5f..d2a8a01 100644
--- a/ATMTextLayer.m
+++ b/ATMTextLayer.m
@@ -9,6 +9,7 @@
  *	https://github.com/atomton/ATMHud
  */
 
+#import <UIKit/UIKit.h>
 #import "ATMTextLayer.h"
 
 @implementation ATMTextLayer
