--- cpCollision.c	2015-01-08 12:37:50.000000000 -0600
+++ cpCollision2.c	2015-01-08 12:39:13.000000000 -0600
@@ -158,13 +158,14 @@
 	cpVect n;
 };
 
+/*
 static inline struct Edge
 EdgeNew(cpVect va, cpVect vb, cpHashValue ha, cpHashValue hb, cpFloat r)
 {
 	struct Edge edge = {{va, ha}, {vb, hb}, r, cpvnormalize(cpvperp(cpvsub(vb, va)))};
 	return edge;
 }
-
+*/
 static struct Edge
 SupportEdgeForPoly(const cpPolyShape *poly, const cpVect n)
 {
